mantidproject/mantid
# Move model creation functions to a TestHelpers subdirectory

This PR moves the ModelCreationHelper files to a new TestHelpers subdirectory. This makes it clear that these functions are only for testing.

The larger changes to the top level CMakeLists.txt are from running cmake_format.

Refs #26535

**To test:**

- Code review and ensure unit tests still pass.
- Check that the ISIS Reflectometry GUI still opens.

*This does not require release notes* because **it concerns file tidying only**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
